client/containers/Pub/SpubHeader/SpubHeaderToolBar/SubmitDialog.tsx

```diff
@@ -24,7 +27,15 @@ const SubmitDialog = (props: Props) => {
 					id: props.submission.id,
 					status: 'pending' as SubmissionStatus,
 				})
-				.then((submissionRes) => setUpdatedSubmission(submissionRes))
+				.then((submissionRes) => {
+					setUpdatedSubmission(submissionRes);
+					updateSubmissionState(({ submission }) => ({
+						submission: {
```

Comment:
This is interesting.

We can destructure an object and update the values that may come from the restructured object as well